webpack.config.extensions.js

```diff
@@ -131,12 +130,6 @@ function getWebpackConfig() {
 			new webpack.IgnorePlugin( /^\.\/locale$/, /moment$/ ),
 			...SassConfig.plugins( { cssFilename, minify: ! isDevelopment } ),
 			new DuplicatePackageCheckerPlugin(),
-			new CopyWebpackPlugin( [
-				{
-					from: presetPath,
-					to: 'index.json',
-				},
-			] ),
```

Comment:
FYI @jeffersonrabb, you can most likely remove this from Newspack as well, I think I saw it there.